sdcm/cluster.py

```diff
@@ -1660,6 +1660,13 @@ def stop_nemesis(self, timeout=10):
         self.nemesis_threads = []
         self.log.debug('Stop nemesis end')
 
+    def _experimental(self):
+        for cmd in ('stress_cmd', 'stress_cmd_1'):
+            stress_cmd = self.params.get(cmd)
+            if stress_cmd and 'counter_' in stress_cmd:
+                return True
+        return False
+
 
```

Comment:
Rethinking about it, I think we said we don't need this check.
This is relevant only to 1.7 and we said that we can explicitly mention it in the yaml file.